src/components/Sections/VideoSection/index.tsx

```diff
@@ -0,0 +1,54 @@
+import BaseComponent from "@/components/BaseComponent";
+import LineSeparator from "@/components/LineSeparator";
+import Headline from "@/components/Headline";
+import { VideoSectionProps } from "@/types/sections";
+import { Component, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "VideoSection",
+})
+class VideoSection extends BaseComponent<VideoSectionProps> {
+  @Prop() youtubeId!: VideoSectionProps["youtubeId"];
+  @Prop() width: VideoSectionProps["width"];
+  @Prop() height: VideoSectionProps["height"];
+  @Prop() title: VideoSectionProps["title"];
+  @Prop() description: VideoSectionProps["description"];
+  @Prop() parameters: VideoSectionProps["parameters"];
+  @Prop() youtubeUrl: VideoSectionProps["youtubeUrl"];
+
+  properties() {
+    return this.parameters
+      ? this.parameters
+          .map((p, index) => (index > 0 ? "&" : "?") + p.param + "=" + p.value)
+          .join("")
+      : "";
+  }
+
+  render() {
+    const videoWidth = this.width ? this.width : 560;
+    const url = this.youtubeUrl
```

Comment:
We should make sure that `this.youtubeUrl` ends with a slash, otherwise we cannot build a valid URL for our IFrame below.

Reply:
Done.